target_chains/solana/examples/send_usd/app/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "app",
```

Comment:
We should add this to lerna like the evm example

Reply:
Done.